build.fsx

```diff
@@ -420,7 +385,7 @@ Target "HelpDocs" <| fun _ ->
 //--------------------------------------------------------------------------------
 
 // build dependencies
-"Clean" ==> "AssemblyInfo" ==> "RestorePackages" ==> "UpdateDependencies" ==> "Build" ==> "CopyOutput" ==> "BuildRelease"
```

Comment:
Yes, otherwise the build script would update all our dependencies to the latest version on every build, now it sticks with the current one and everyone can use which version they want in there implementation if it's >= the current one.